tests/container/BUILD

```diff
@@ -1,5 +1,499 @@
+         "digest": "sha256:f29c3d10359dd0e6d0c11e4f715735b678c0ab03a7ac4565b4b6c08980f6213b"
+      }
+   ]
+}""",
+    file = "@new_alpine_linux_armv6//image:index.json",
+)
+
+file_test(
+    name = "new_distroless_base_test_index_json",
+    content = """{
+   "schemaVersion": 2,
+   "manifests": [
+      {
+         "mediaType": "application/vnd.docker.distribution.manifest.v2+json",
+         "size": 737,
+         "digest": "sha256:edc3643ddf96d75032a55e240900b68b335186f1e5fea0a95af3b4cc96020b77"
+      }
+   ]
+}""",
+    file = "@new_distroless_base//image:index.json",
+)
+
+file_test(
+    name = "new_alpine_linux_armv6_test_oci_layout",
+    content = """{
+    "imageLayoutVersion": "1.0.0"
+}""",
+    file = "@new_alpine_linux_armv6//image:oci-layout",
+)
+
+file_test(
+    name = "new_distroless_base_test_oci_layout",
+    content = """{
+    "imageLayoutVersion": "1.0.0"
+}""",
+    file = "@new_distroless_base//image:oci-layout",
+)
+
+# Merely testing for the presence of the pulled image.tar for now.
+file_test(
+    name = "new_alpine_linux_armv6_tar_test_image_tar",
+    file = "@new_alpine_linux_armv6_tar//image:image.tar",
+    regexp = ".*",
+)
+
+file_test(
+    name = "new_distroless_base_both_test_image_tar",
+    file = "@new_distroless_base_both//image:image.tar",
+    regexp = ".*",
+)
+
+file_test(
+    name = "new_distroless_base_both_test_oci_layout",
+    content = """{
+    "imageLayoutVersion": "1.0.0"
+}""",
+    file = "@new_distroless_base_both//image:oci-layout",
+)
+
+# To test the new container load binary
+file_test(
+    name = "new_pause_tar_test_index_json",
+    content = """{
+   "schemaVersion": 2,
+   "manifests": [
+      {
+         "mediaType": "application/vnd.docker.distribution.manifest.v2+json",
+         "size": 586,
+         "digest": "sha256:22e0bb47f6dd1938d7c30febe82ac37e99907fd8c76421ab87829ad6a3636bf9"
+      }
+   ]
+}""",
+    file = "@new_pause_tar//image:index.json",
+)
+
+file_test(
+    name = "new_pause_tar_test_oci_layout",
+    content = """{
+    "imageLayoutVersion": "1.0.0"
+}""",
+    file = "@new_pause_tar//image:oci-layout",
+)
+
+create_banana_directory(
+    name = "banana_directory",
+)
+
+container_image(
+    name = "stripped_directory_name",
+    data_path = "banana",
+    files = [":banana_directory"],
+)
+
+container_test(
+    name = "stripped_directory_name_test",
+    configs = ["//tests/container/configs:stripped_directory_name.yaml"],
+    image = ":stripped_directory_name",
+)
+
+container_bundle(
+    name = "three_images_bundle",
+    images = {
+        "localhost:5000/image0:latest": "//testdata:base_with_entrypoint",
+        "localhost:5000/image1:latest": "//testdata:link_with_files_base",
+        "localhost:5000/image2:latest": "//testdata:with_double_env",
+    },
+)
+
+docker_push_all(
+    name = "test_docker_push_three_images_bundle",
+    bundle = ":three_images_bundle",
+)
+
+[container_test(
+    name = "test_dockerfile_image_%s" % driver,
```

Comment:
This is testing dockerfile_image from contrib. Probably should be moved to tests/contrib, together with the testing configs.

Reply:
done